code/modules/organs/organ_external.dm

```diff
@@ -821,6 +821,9 @@ Note that amputating the affected organ does in fact remove the infection from t
 			//Throw organs around
 			var/randomdir = pick(cardinal)
 			step(organ, randomdir)
+			
+		if(istype(src, /datum/organ/external/head))
```

Comment:
rather than istyping src, can you put this in head's droplimb?

Reply:
Done.